datafusion/core/tests/integration_tests/schema_adapter_integration_tests.rs

```diff
@@ -258,3 +259,187 @@ fn test_schema_adapter_preservation() {
     // Verify the schema adapter factory is present in the file source
     assert!(config.source().schema_adapter_factory().is_some());
 }
+
```

Comment:
this test is moved without change from `datafusion/core/tests/test_adapter_updated.rs`